src/shared/components/CartModalWindow.tsx

```diff
@@ -0,0 +1,321 @@
+            </Typography>
+            <IconButton
+              sx={{
+                color: FILTER_COLORS.BUTTON_RESET_FILTER,
+                padding: 0,
+                position: "absolute",
+                top: 10,
+                right: 10,
+              }}
+            >
+              <CloseIcon
+                onClick={handleCloseCartModalWindow}
+                sx={{
+                  height: "20px",
+                  width: "20px",
+                  padding: 0,
+                }}
+              />
+            </IconButton>
+          </Box>
+          <List
+            sx={{
+              padding: 0,
+              overflowY: "auto",
+              flex: 1,
+            }}
+          >
+            {cartItemDetails.map((cartItem: any, index: any) => (
+              <ListItem key={index} disablePadding>
+                <Box
+                  sx={{
+                    borderBottom: `1px solid ${BASE_COLORS.DEFAULT_BLUE}`,
+                    cursor: "default",
+                    margin: "0 10px",
+                    paddingTop: "10px",
+                    position: "relative",
+                    width: "100%",
+                  }}
+                >
+                  <IconButton
+                    sx={{
+                      padding: 0,
+                      position: "absolute",
+                      top: 10,
+                      right: 10,
+                      "&:hover": {
+                        "& .MuiSvgIcon-root": {
+                          color: FILTER_COLORS.BUTTON_RESET_FILTER,
+                        },
+                      },
+                    }}
+                  >
+                    <DeleteIcon
+                      onClick={() => handleDeleteItem(cartItem.tireId)}
+                      sx={{
+                        height: "20px",
+                        width: "20px",
+                        padding: 0,
+                        transition: "all 0.1s ",
+                      }}
+                    />
+                  </IconButton>
+                  <Box display={"flex"} flexDirection={"column"} mb={1}>
+                    <Link
+                      href={`/item?id=${cartItem.article.toString()}`}
+                      color={"inherit"}
+                      sx={{ textDecoration: "none", display: "inline" }}
+                    >
+                      <Typography
+                        variant="subtitle2"
+                        fontWeight={600}
+                        fontFamily={FONTS.BOLD_TEXT_FAMILY}
+                        mb={1}
+                        p={"0 15px"}
+                      >
+                        {cartItem.fullName}
+                      </Typography>
+                    </Link>
+                    <Box display={"flex"} gap={"4%"}>
+                      <img
+                        src={cartItem.image}
+                        alt={cartItem.name}
+                        style={{ width: "100px", height: "100px" }}
```

Comment:
create a `styled` Box component instead of using base HTML tags

Reply:
done